### PR TITLE
feat(analytics): add Matomo support

### PR DESCRIPTION
**Pull request by ludovicm67**
This PR adds support for Matomo, as mentionned in #6411.

I added some comments in the `config.js` file to illustrate required settings to configure Matomo.

The next step will be to make it configurable using environment variables when using the Docker image (https://github.com/jitsi/docker-jitsi-meet/issues/529).

### CODE REVIEW

**Comment by ludovicm67** [comment]
> Hi, thanks for your contribution!
> If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

Already done.

File: webpack.config.js [hunk 1]
```diff
@@ -217,6 +217,12 @@ module.exports = [
         },
         performance: getPerformanceHints(5 * 1024)
     }),
+    Object.assign({}, config, {
```
Comment (saghul) [inline webpack.config.js:220]: I'd say Matomo is relevant enough to warrant its inclusion by default, so no need to create a separate bundle. See how we do for Amplitude.

File: Makefile [hunk 2]
```diff
@@ -46,6 +46,8 @@ deploy-appbundle:
 		$(OUTPUT_DIR)/analytics-ga.js \
 		$(BUILD_DIR)/analytics-ga.min.js \
 		$(BUILD_DIR)/analytics-ga.min.map \
+		$(BUILD_DIR)/analytics-matomo.min.js \
```
Comment (saghul) [inline Makefile:49]: Let's not do a separate bundle, see my message below.

File: config.js [hunk 3]
```diff
@@ -368,13 +368,18 @@ var config = {
...
+        // matomoSiteID: '42',
+
         // The Amplitude APP Key:
         // amplitudeAPPKey: '<APP_KEY>'
 
         // Array of script URLs to load as lib-jitsi-meet "analytics handlers".
         // scriptURLs: [
-        //      "libs/analytics-ga.min.js", // google-analytics
-        //      "https://example.com/my-custom-analytics.js"
+        //     "libs/analytics-ga.min.js",     // google-analytics
```
Comment (saghul) [inline config.js:380]: Drop this too.

**Comment by saghul** [comment]
Jenkins please test this please.

**Pull request merged**